Plots/Scatter/NCL_scatter_6.py

```diff
@@ -0,0 +1,72 @@
+"""
+NCL_scatter_6.py
+===============
+This script illustrates the following concepts:
+   - Drawing a scatter plot with markers of different colors and sizes
+   - Drawing outlined and filled markers on a polar map plot
+   - Generating dummy data using "random"
+   - Changing the marker colors on a polar map plot
+   - Changing the marker sizes on a polar map plot
+   - Turning off y-axis labels
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/scatter_6.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/scatter_6_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import matplotlib.pyplot as plt
+
+###############################################################################
+# Create dummy data:
+
+numpoints = 100
+lon = np.random.randint(0, 360, numpoints)
+lat = np.random.randint(5, 90, numpoints)
+
+###############################################################################
+
+# Generate a figure
+fig = plt.figure(figsize=(8, 8))
+
+# Create axis with a polar projection
+ax = fig.add_subplot(111, projection='polar')
+ax.set_ylim([0, 90])
+
+# Change orientation of projection
+ax.set_theta_zero_location("S")
+
+# Create array of marker colors
+colors = ("limegreen", "orange", "green", "red", "yellow", "purple",
+          "blue", "red", "brown", "crimson", "skyblue")
+
+# Create array of marker sizes
+bins = np.linspace(100, 2000, 10)
+
+# Plot all points
+# longitude points must be transformed to be plotted on polar projection
```

Comment:
Can you elaborate on this transformation? You're going from degrees to radians, but make sure you specify that in the comments.